### ISSUE TITLE
👨‍💻 is more relevant than 💻 for code contribution

### ISSUE THREAD

**Issue by yedhrab** [all-contributors/all-contributors#363]
**Is your feature request related to a problem? Please describe.**
👨‍💻 emoji is more relevant than 💻 emoji for code contribution

**Describe the solution you'd like**
💻 -> 👨‍💻 👩‍💻 (Please convert pc emoji to code emoji)

**Describe alternatives you've considered**
- For famale: 👩‍💻 
- For male: 👨‍💻

**Comment by Berkmann18** [comment]
TL;DR: That approach has more cons and than pros so not worth it imo.

The problem with :man_technologist: and :woman_technologist: is that they represent a specific sex (which could be discriminatory to some contributors).
Also, I'm not sure if you read the [point 3 of the specification](https://allcontributors.org/docs/en/specification) but it clearly mentions:
> This specification is, by definition, inclusive of all contributions
> [...] Instead, projects should use indicators of effort within a list that remains inclusive of all contributors to the project at any level of effort

Re: (tl;dr: not using a sex specific emoji)
https://github.com/all-contributors/all-contributors-cli/issues/111#issuecomment-454388086
https://github.com/all-contributors/all-contributors/pull/49/files

:computer: is imo (and I think most of the team + contributors/users) clear enough perfectly relevant to a `code` contribution.

Also, I don't think adding variants for the _same_ contribution type is a good idea as it will lead to a bigger set of contribution types so a more complex CLI and bot which may identify the gender of contributors (a good and bad thing) while excluding those who identify differently.

**Comment by yedhrab** [comment]
I got it, you're right. also thank you for detailed explanation :)

**Comment by Berkmann18** [comment]
No problem :grin:.
